src/docs/product/issues/states-triage/index.mdx

```diff
@@ -8,82 +8,62 @@ To help organize issues for you and your team, you can take actions such as assi
-
-### Link Issues
-
-Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/sentry-basics/guides/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [global integrations](/product/integrations/integration-platform/#public-integrations) or [internal integrations](/product/integrations/integration-platform/#internal-integrations) are searchable using the terms `is:linked` and `is:unlinked`.
-
-### Resolve
-
-Resolve an issue when it's fixed or you don't expect it to happen again. You can do this manually or by [including the issue ID in a commit](/product/integrations/gitlab/#resolve-via-commit-or-pull-request).
-<Note>
+- A new issue is created; it's labeled "New Issue"
+- A resolved issue re-occurs; it's labeled "Regression"
+- An ignored issue meets its ignore conditions; it's labeled "Unignored"
 
-The Auto Resolve setting defines the amount of time an issue can go without being seen again (that is, no new events on that issue) before it's automatically resolved. To check if this has been defined for a project, go to **[Project] > Settings > General Settings** and check the Event Settings section.
+For-review issues are a sub-set of all unresolved issues, and you can view these issues in the "For Review" tab of the **Issues** page. By default, they are filtered to display issues relevant to you; that is, issues that are assigned to you or suggested for you.
 
-</Note>
+![For Review tab of the Issues page.](for-review-page-docs.png)
 
-### Ignore
+When you mark a for-review issue as reviewed, [sentry.io](https://sentry.io) removes the reason label, and the issue remains in the Unresolved state.
 
-Ignore an issue **permanently** (the default action when you press the button) if it's just noise and you'll likely never care about it. Or ignore it until a condition is met, at which point it becomes Unresolved. You can also set an issue to Ignore if you're working on the issue if you don't want to keep getting alerts.
+Resolving or ignoring an issue implicitly marks the issue reviewed.
 
-### Delete
+To keep the list of issues in the "For Review" tab fresh, [sentry.io](https://sentry.io) automatically marks issues as reviewed if they haven't already been reviewed after seven days.
 
-Delete an issue to remove it from the system. A new issue is created if a new instance of the issue (a new event) occurs.
+The image below shows how issues move through different states with the applicable labels.
 
-### Delete and Discard
+![Worklow for triaging issues](issue-workflow.png)
```

Comment:
Future: might make this a short video?

Reply:
I think that's a really good option, yeah.